pkg/ipam/types/stats.go

```diff
@@ -0,0 +1,15 @@
+package types
```

Comment:
Note that there are some linter errors in this file

Reply:
fixed, thanks!